region/crateria/central/Climb Supers Room.json

```diff
@@ -1609,6 +1654,7 @@
           ]}
         ]}
       ],
+      "flashSuitChecked": true,
```

Comment:
This might be complex or might not.  It should be roughly as hard as the damageboost for fumbling the flash suit.